src/server/svr_chk_owner.c

```diff
@@ -390,66 +390,94 @@ authenticate_user(struct batch_request *preq, struct connection *pcred)
  *		this checks the most conditions common to most job batch requests.
  *		It also returns a pointer to the job if found and the tests pass.
  *		If the request is for a single subjob or a range of subjobs (of an
- *		Job Array),  the return job pointer is to the parent Array Job.
+ *		Job Array), alsoe return pointer to the parent Array Job.
```

Comment:
correct the typo here.

Reply:
fixed in rc 10 https://github.com/PBSPro/pbspro/pull/590/commits/1631a47b00ac805440c976a0ee8fb82ec39c2284